### DISCUSSION TITLE
[Bug]: o1mini not adhering to backticks

### DISCUSSION THREAD

**Discussion by paradroidlabs** [pnd280/complexity#21] [General]
### Version

0.0.3.14

### Describe the bug

https://www.perplexity.ai/search/perform-recursive-peer-review-BSMM8t6vQZKjF6Q2PtpU.Q#0
Output is "within" scratchpad canvas. 

### Steps to reproduce

random issue with o1mini, not easily repeatable without blowing thru o1mini use.

### Bug confirmation

- [X] I am not able to reproduce the issue with the extension disabled
- [X] I have re-installed the extension and the issue persists

### Expected behavior

![Recording 2024-10-16 at 21 59 41](https://github.com/user-attachments/assets/884ea4fa-f927-4e96-b43f-254aef7c6ccd)

### Browser

Chrome/latest.

### Additional context

posting here over discord

**Comment by pnd280** [comment]
i don't think the current scratchpad structure is working with o1, considering it's already doing its own reasoning
